war/src/main/less/abstracts/font.less

```diff
@@ -1,9 +1,12 @@
-@font-family-sans: -apple-system, BlinkMacSystemFont, "Segoe UI", Roboto, "Noto Sans", Oxygen, Ubuntu, Cantarell, "Fira Sans", "Droid Sans", "Helvetica Neue", Arial, sans-serif, "Apple Color Emoji", "Segoe UI Emoji", "Segoe UI Symbol";
-
-@font-size-base: 1rem; // 16px
-@font-size-sm: 0.875rem; // 14px
-@font-size-xs: 0.75rem; // 12px
-@font-size-monospace: 0.95em;
-
-@line-height-base: 1.5;
-@line-height-heading: 1.2;
+body {
+  // Font related properties
+  --font-family-sans: -apple-system, BlinkMacSystemFont, "Segoe UI", Roboto, "Noto Sans", Oxygen, Ubuntu, Cantarell, "Fira Sans", "Droid Sans", "Helvetica Neue", Arial, sans-serif, "Apple Color Emoji", "Segoe UI Emoji", "Segoe UI Symbol";
```

Comment:
Maybe the fact that the variables are declared here and not in `:root` is causing the fallback issues. Just some brainstorming.

Reply:
Yes you’re right

Reply:
Maybe we should just go ahead and create a big & comprehensive `variables.less` file and be done with it :D